### PR TITLE
Add blog post for standard commitment badge

### LINKED ISSUE #445: Blog post about commitment badge
Write a blog post about the commitment badge and how to use it. Show an example of a codebase (which is not ours) that is using it.

### PR DESCRIPTION
**Pull request by Ainali**
Fixes #445

Aims to be published on Tuesday.

### CODE REVIEW

**Review by ericherman — APPROVED** [review]

File: _posts/2023-09-12-standard-for-pubic-code-commitment-badge.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+author: Jan Ainali
+type: blogpost
+excerpt: How to display a codebase's commitment to the Standard for Public Code
+categories:
+  - News
+---
+
+# Standard for Public Code commitment badge
+
+As the Standard for Public Code imposes a large number of requirements on codebases that commit to meet it, and subsequently, on all future contributions made to a codebase, it is very helpful for potential contributors to discover as early as possible in their collaboration process that it applies for them too.
```
Comment (ElenaFdR) [inline _posts/2023-09-12-standard-for-pubic-code-commitment-badge.md:13]:
```suggestion
As the Standard for Public Code imposes a large number of requirements on codebases that commit to meet it, and subsequently, on all future contributions made to a codebase, it is very helpful for potential contributors to discover as early as possible in their collaboration process that it applies to them too.
```

File: _posts/2023-09-12-standard-for-pubic-code-commitment-badge.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+![Standard for Public Code commitment badge](https://raw.githubusercontent.com/publiccodenet/standard/develop/assets/standard-for-public-code-commitment.svg)
+
+By using the badge in a repository, and linking it to that codebase's commitment to meet it fully, it should be clear to anyone that sees it that the codebase is serious about collaboration and what the expectations are for new contributions.
+
+We are happy to see that this badge is starting to be used, for example in the Swedish Agency for Digital Government's [Open Source Project Template](https://github.com/diggsweden/open-source-project-template#open-source-project-template) and [jitsi-outlook](https://github.com/diggsweden/jitsi-outlook/blob/main/CONTRIBUTING.adoc#standard-for-public-code) repositories.
+
+If you too want to show your commitment to meet the Standard for Public Code in a codebase you maintain, it's fairly straightforward to add the badge to your repository, as explained in the  [README for the Standard for Public Code](https://github.com/publiccodenet/standard#applying-the-standard-for-public-code-to-your-codebase).
```
Comment (ElenaFdR) [inline _posts/2023-09-12-standard-for-pubic-code-commitment-badge.md:23]:
```suggestion
If you too want to show your commitment to meet the Standard for Public Code in a codebase you maintain, it's fairly straightforward to add the badge to your repository, as explained in the [README for the Standard for Public Code](https://github.com/publiccodenet/standard#applying-the-standard-for-public-code-to-your-codebase).
```

File: _posts/2023-09-12-standard-for-pubic-code-commitment-badge.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+We link to codebases that we are aware of having made a commitment like this in our list over at [standard-compliant.publiccode.net](https://standard-compliant.publiccode.net/#public-commitment), whether they have added the badge or not.
+If you want your codebase to show up in that list, please email a link to a public commitment to meet the Standard for Public Code to [standard@publiccode.net](mailto:standard@publiccode.net) and we will add it.
+Or, if you like, you can also make a pull request directly to [the standard-compliant.publiccode.net repository](https://github.com/publiccodenet/standard-compliant).
```
Comment (ElenaFdR) [inline _posts/2023-09-12-standard-for-pubic-code-commitment-badge.md:27]:
```suggestion
Or, if you like, you can also make a pull request directly to the [standard-compliant.publiccode.net repository](https://github.com/publiccodenet/standard-compliant).
```

**Review by ElenaFdR — COMMENTED** [review]
Tiny nitpicky changes, but otherwise very clear (and exciting news!)

**Review by ericherman — APPROVED** [review]

**Pull request merged**